manubot/process/util.py

```diff
@@ -223,8 +218,22 @@ def load_variables(args) -> dict:
     if thumbnail_url:
         variables["manubot"]["thumbnail_url"] = thumbnail_url
 
+    # Update variables with metadata.yaml pandoc/manubot dicts
+    for key in "pandoc", "manubot":
+        dict_ = metadata.pop(key, {})
+        if not isinstance(dict_, dict):
+            logging.warning(
+                f"load_variables expected metadata.yaml field {key!r} to be a dict."
+                f"Received a {dict_.__class__.__name__!r} instead."
+            )
+            continue
+        variables[key].update(dict_)
+
+    # Update variables with uninterpreted metadata.yaml fields
+    variables.update(metadata)
+
     # Update variables with user-provided variables here
-    user_variables = read_jsons(args.template_variables_path)
+    user_variables = read_variable_files(args.template_variables_path)
```

Comment:
Setting `pandoc` or `manubot` in a `--template-variables-path` input would overwrite the entirety of those dicts (everything set by Manubot and by metadata.pandoc / manubot.manubot). This could be risky. I will think about whether we need a warning or behavior change.

Reply:
434c7255ab179b9a8a62d077bc5bc6688bbddbb7: now if you overwrite any metadata values (including `manubot` / `pandoc`) with user variables, there is a warning.